mantle/kola/tests/ostree/basic.go

```diff
@@ -47,7 +47,7 @@ func init() {
 		Description: "Verify the ostree remote functions work.",
 		Flags:       []register.Flag{register.RequiresInternetAccess}, // need network to contact remote
 		FailFast:    true,
-		Tags:        []string{"ostree"},
+		Tags:        []string{"ostree", "needs-internet"},
```

Comment:
Note this is redundant with the `RequiresInternetAccess` flag. I think we need to move over anything still using that flag to the tag approach (some other internal tests already did that, see e.g. `podman.network-single`) and just kill that flag.

From a `git grep`, there aren't that many left. Want to tackle that in this PR? Cool if not, but it's probably worth a comment at least since from the point of view of this test it's not obvious why we have both.

Reply:
Also, it's probably better to use `kola.NeedsInternetTag` for consistency?

Reply:
ok I tried to implement this now.